.github/dependabot.yml

```diff
@@ -7,10 +7,10 @@ updates:
     labels:
       - dependencies
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
For whatever reason, there have been a ton of version updates lately and it's kind of too much.   Would really like to get the dependabot back to being able to merge into `dev` without human supervision if all the CI passes.